.gitignore

```diff
@@ -6,7 +6,6 @@ packages/viewer/dist
 *.crt
 .nyc_output
 coverage/
-.vscode
```

Comment:
removing this would make commiting the launch.json files possible. I think we should not allow that.

Reply:
Why not? Would result in better DX imo, considering that each dev wouldn't have to come up with his own launch config, but could use one of the standardized ones

Reply:
if possible i'd lock this folder to be only editable by the speckle team, however

Reply:
for specific testing purposes i usually change some variables, ie bake in the regex, for the test runner, or override some environment variables just for a specific startup config. i kinda like doing that, since it allows me to run different scenarios. These are usually not that applicable to others and sometimes might be os specific too.

The npm scripts part of vscode is clever enough to pick up all the subpackage scripts, which works perfectly if you don't want to customize stuff. I'd guide newcommers towards that and leave the launch.json as a per machine option open.
![image](https://user-images.githubusercontent.com/57442769/158388415-d472db3d-b5c9-46f8-a152-942c272e008c.png)

Reply:
@gjedlicska I think we should discuss this further in chat, I think that in most scenarios you should be able to get away with using standard debug configs.